tests/foreman/destructive/test_ldap_authentication.py

```diff
@@ -854,3 +877,202 @@ def test_positive_negotiate_logout(
     with pytest.raises(CLIReturnCodeError) as context:
         parametrized_enrolled_sat.cli.Host.list()
     assert AUTH_FAILED_MSG in context.value.message
+
+
+@pytest.mark.parametrize(
+    'parametrized_enrolled_sat,user_not_exists',
+    [('IDM', settings.ipa.user), ('AD', f'{settings.ldap.username}@{settings.ldap.realm.lower()}')],
```

Comment:
This is failing github checks. Since the checks here just use the template files (with some minor changes), it will use whatever the default value is in the template file. If we look at this value here:
https://github.com/SatelliteQE/robottelo/blob/master/conf/ldap.yaml.template#L10
We see that it has no value set, so yaml considers that to be `null/None`.
Additionally, the validator for this field doesn't set a default, it just makes sure that the field exists.
https://github.com/SatelliteQE/robottelo/blob/master/robottelo/config/validators.py#L195

You have a couple ways around this.

1. set some kind of dummy string value in the template or in the validator
2. change this lookup to set a default value, like this suggestion.

```suggestion
    [('IDM', settings.ipa.user), ('AD', f'{settings.ldap.username}@{settings.ldap.get('realm', '').lower()}')],
```

Reply:
The second suggested solution doesn't work, I'll try the template way.
```

==================================== ERRORS ====================================
____ ERROR collecting tests/foreman/destructive/test_ldap_authentication.py ____
tests/foreman/destructive/test_ldap_authentication.py:886: in <module>
    ('AD', f'{settings.ldap.username}@{settings.ldap.get("realm", "").lower()}'),
E   AttributeError: 'NoneType' object has no attribute 'lower'
=========================== short test summary info ============================
ERROR tests/foreman/destructive/test_ldap_authentication.py - AttributeError: 'NoneType' object has no attribute 'lower'
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======== 5621/6075 tests collected (454 deselected), 1 error in 24.08s =========
```